File: conductor-car-dealer-demo/car-dealer/src/main/java/com/github/phillbarber/conductor/order/CheckOrderIsValidWorker.java
package com.github.phillbarber.conductor.order;

import com.netflix.conductor.client.worker.Worker;
import com.netflix.conductor.common.metadata.tasks.Task;
import com.netflix.conductor.common.metadata.tasks.TaskResult;

import java.util.HashMap;

public class CheckOrderIsValidWorker implements Worker {

    private String taskName;

    public CheckOrderIsValidWorker(String taskName) {
        this.taskName = taskName;
    }

    @Override
    public String getTaskDefName() {
        return taskName;
    }

    public TaskResult execute(Task task)  {
        TaskResult result = new TaskResult(task);
        result.setStatus(TaskResult.Status.COMPLETED);

        HashMap<String, Object> objectObjectHashMap = new HashMap<>();
        objectObjectHashMap.put("isValid", true);
        result.setOutputData(objectObjectHashMap);

        return result;
    }
}

File: conductor-car-dealer-demo/src/main/kotlin/CheckOrderIsValidWorker.kt
import com.netflix.conductor.client.worker.Worker
import com.netflix.conductor.common.metadata.tasks.Task
import com.netflix.conductor.common.metadata.tasks.TaskResult

class CheckOrderIsValidWorker(val taskName: String) : Worker {

    override fun getTaskDefName(): String {
        return this.javaClass.name
    }

    override fun execute(task: Task?): TaskResult {
        val result = TaskResult(task)
        result.status = TaskResult.Status.COMPLETED

        result.outputData["isvalid"] = true

        return result
    }
}